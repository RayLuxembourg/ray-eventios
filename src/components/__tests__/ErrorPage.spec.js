import React from "react";
import { ErrorPage } from "../";
import { shallow } from "enzyme";

// Enzyme.configure({ adapter: new Adapter() });
describe("Attendess component test", () => {
  test("renders without crashing", () => {
    const Error = shallow(<ErrorPage />);
    expect(Error).toMatchSnapshot();
  });
});
