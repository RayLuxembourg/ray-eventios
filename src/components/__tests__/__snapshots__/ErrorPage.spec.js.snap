// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attendess component test renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ErrorPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Headline
          subtitle="Seems like Darth Vader just hits our website and drops it down.
                        Please press the refresh button and everything should be fine again."
          title="404 Error - page not found."
/>,
        <Button
          color="dark"
          content="REFRESH"
          onClick={[Function]}
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "subtitle": "Seems like Darth Vader just hits our website and drops it down.
                        Please press the refresh button and everything should be fine again.",
          "title": "404 Error - page not found.",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "color": "dark",
          "content": "REFRESH",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Headline
            subtitle="Seems like Darth Vader just hits our website and drops it down.
                        Please press the refresh button and everything should be fine again."
            title="404 Error - page not found."
/>,
          <Button
            color="dark"
            content="REFRESH"
            onClick={[Function]}
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "subtitle": "Seems like Darth Vader just hits our website and drops it down.
                        Please press the refresh button and everything should be fine again.",
            "title": "404 Error - page not found.",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "color": "dark",
            "content": "REFRESH",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
